### PR TITLE
Database functions

### PR DESCRIPTION
**Pull request by dsloan92**
Ready to merge. When reviewing please pull branch to ensure that the functions are working as expected. My computer, for whatever reason, is not working.

### CODE REVIEW

**Comment by kellerflint** [comment]
- Data type required for @param in database.php (line 68,69)
- Tutor table's user_id should not AUTO_INCREMENT but it should in the User's table. The Year primary key should also AUTO_INCREMENT
- Personally I would get users/tutors by their id and not by email since a pk is more unique and it won't be hard to get because we'll have it saved to the session once they log in.

Otherwise commenting is awesome and it works on my end

**Review by kellerflint — CHANGES_REQUESTED** [review]
- Data type required for @param in database.php (line 68,69)
- Tutor table's user_id should not AUTO_INCREMENT but it should in the User's table. The Year primary key should also AUTO_INCREMENT
- Personally I would get users/tutors by their id and not by email since a pk is more unique and it won't be hard to get because we'll have it saved to the session once they log in.

Otherwise commenting is awesome and it works on my end

**Comment by dsloan92** [comment]
Thanks for the review Keller. I made the requested changes. I agree about using the userID instead of email, that has been changed.

File: model/database.php [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    function getTutor($year, $userID)
+    {
+        //defining query
+        $sql = "SELECT Tutor.tutor_first, Tutor.tutor_last, User.user_email, Year.year_packet_sent, Year.year_background,
+                    Year.year_reference, Year.year_offer_letter, Year.year_affirmation_disclosures, Year.year_sexual_misconduct,
+                    Year.year_w4, Year.year_handbook_verification, Year.year_ADP, Year.year_i9, Year.year_orientation,
+                    Year.year_placement from Year
+                    JOIN Tutor on Year.user_id = Tutor.user_id
+                    JOIN User on Year.user_id = User.user_id
+                    where User.user_email = ?
```
Comment (kellerflint) [inline model/database.php]: where User.user_id =?

**Review by kellerflint — CHANGES_REQUESTED** [review]

**Pull request merged**